docs/source/llms/langchain/autologging.rst

```diff
@@ -98,6 +98,53 @@ Example Code of LangChain Autologging
     :language: python
 
 
+Tracing LangGraph
+-----------------
+
+MLflow also support automatic tracing for LangGraph, an open-source library from LangChain for building stateful, multi-actor applications with LLMs, used to create agent and multi-agent workflows. To enable auto-tracing for LangGraph, use the same :py:func:`mlflow.langchain.autolog()` function.
+
+.. code-block:: python
+
+    from typing import Literal
+
+    import mlflow
+
+    from langchain_core.messages import AIMessage, ToolCall
+    from langchain_core.outputs import ChatGeneration, ChatResult
+    from langchain_core.tools import tool
+    from langchain_openai import ChatOpenAI
+    from langgraph.prebuilt import create_react_agent
+
+    # Enabling tracing for LangGraph (LangChain)
+    mlflow.langchain.autolog()
+
+    # Optional: Set a tracking URI and an experiment
+    mlflow.set_tracking_uri("http://localhost:5000")
+    mlflow.set_experiment("LangGraph")
+
+
+    @tool
+    def get_weather(city: Literal["nyc", "sf"]):
+        """Use this to get weather information."""
+        if city == "nyc":
+            return "It might be cloudy in nyc"
+        elif city == "sf":
+            return "It's always sunny in sf"
+
+
+    llm = ChatOpenAI(model="gpt-4o-mini")
+    tools = [get_weather]
+    graph = create_react_agent(llm, tools)
+
+    # Invoke the graph
+    result = graph.invoke(
+        {"messages": [{"role": "user", "content": "what is the weather in sf?"}]}
+    )
+
+.. note::
+
+    MLflow does not support other auto-logging features such as model logging, model signatures, for LangGraph. Only traces are logged for LangGraph.
```

Comment:
```suggestion
    MLflow does not support other auto-logging features such as model logging for LangGraph. Only traces are logged for LangGraph.
```
Since signature is part of model logging